mat64/symmetric.go

```diff
@@ -105,7 +105,20 @@ func (s *SymDense) SetRawSymmetric(b blas64.Symmetric) {
 	s.mat = b
 }
 
+// Reset zeros the dimensions of the matrix so that it can be reused as the
+// receiver of a dimensionally restricted operation.
+//
+// See the Reseter interface for more information.
+func (s *SymDense) Reset() {
+	// No change of Stride and N to 0
```

Comment:
I will add commit that changes all three Reset comment to this PR. The wording needs thought.